scripts/jenkins/ci.suse.de/openstack-mkcloud-pr-trigger.xml

```diff
@@ -136,7 +136,7 @@ for repo in ${crowbar_repos[@]}; do
 for github_pr in $($ghs -r crowbar/$repo -a $action); do
         # Skip any that we have tried before
         test -d /srv/www/htdocs/mkcloud/$repo:$github_pr &amp;&amp; continue
-        ./crowbar-testbuild.py $repo $github_pr
+        ./crowbar-testbuild.py $repo ${github_pr//:/ }
```

Comment:
I disagree.. this makes copy&pasting a PR identifier unnecessarily hard, and serves no value at all (its still positional arguments).